docs/application/web/getting-started/wearable-watch/first-app-watch.md

```diff
@@ -260,7 +260,7 @@ To run the application on the emulator:
 
         ![Application running in the emulator](media/emulator_running_watch_ww.png)
 
-        > **Note** 
+        > **Note**  
 		> If the emulator display has switched off, you cannot see the application launch. To switch the display on, click the **Power** key (in the lower-right corner of the emulator).
 
     3.  Change the watch face.
```

Comment:
Don't we need <a name="watchface"></a> here?

Reply:
Done